server/serverMethods.js

```diff
@@ -655,6 +655,9 @@ function ServerMethods(aLogLevel, aModules) {
     var body = aReq.body;
     var phoneNumber = body.phoneNumber;
     var googleIdToken = body.googleIdToken;
+    if (!tbConfig.enableSip) {
```

Comment:
I guess if it's not enabled we shouldn't be getting requests here but as they say: better safe than unhandled exception.